jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -87,17 +87,36 @@ public void apply(Project project) {
     Task buildImageTask =
         project
             .getTasks()
-            .create(BUILD_IMAGE_TASK_NAME, BuildImageTask.class)
+            .create(
+                BUILD_IMAGE_TASK_NAME,
+                BuildImageTask.class,
+                task -> {
+                  task.setGroup("Jib Docker");
+                  task.setDescription("Builds a container image.");
+                })
             .setJibExtension(jibExtension);
     Task buildDockerTask =
         project
             .getTasks()
-            .create(BUILD_DOCKER_TASK_NAME, BuildDockerTask.class)
+            .create(
+                BUILD_DOCKER_TASK_NAME,
+                BuildDockerTask.class,
+                task -> {
+                  task.setGroup("Jib Docker");
+                  task.setDescription(
+                      "Builds a container image and exports to the default Docker daemon.");
```

Comment:
[Since it's possible to configure the Docker daemon to use](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#dockerclient-closure), we can probably leave out the word "default" (maybe just go with "Builds a container image to a Docker daemon.")

Reply:
[done](https://github.com/aivantsov/jib/commit/9c2ce6c918c8d8c041a2f8838a7fb05ed7aba982)